keps/sig-network/2091-cluster-network-policy/README.md

```diff
@@ -434,13 +435,13 @@ type ClusterNetworkPolicyPeer struct {
 }
 
 const (
-	// RuleActionAuthorize is the highest priority allow rules which enables admins to provide exceptions to deny rules.
-	RuleActionAuthorize RuleAction = "Authorize"
+	// RuleActionEmpower is the highest priority allow rules which enables admins to provide exceptions to deny rules.
```

Comment:
maybe reword this to remove "allow" since its not a allow but more of a bypass now..

Reply:
Done